### PR TITLE
Split sqs queue policy supplier

### PR DESCRIPTION
**Pull request by lotoussa**
| Q                 | A
| ----------------- | ---
| 🐛 Bug fix?       | no
| 🚀 New feature?   | no
| ⚠ Deprecations?   | no
| ❌ BC Break       | no
| 🔗 Related issues | #749 
| ❓ Documentation  | no

branch created from 750 split sqs queue supplier.

### CODE REVIEW

File: pkg/remote/aws/sqs_queue_policy_enumerator.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import (
+	"github.com/cloudskiff/driftctl/pkg/remote/aws/repository"
+	remoteerror "github.com/cloudskiff/driftctl/pkg/remote/error"
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+
+	awssdk "github.com/aws/aws-sdk-go/aws"
+)
+
+type SqsQueuePolicyEnumerator struct {
```
Comment (wbeuil) [inline pkg/remote/aws/sqs_queue_policy_enumerator.go:12]: Can you change everywhere to `SQS` and not `Sqs`. Thanks !

File: pkg/remote/sqs_scanner_test.go [hunk 2]
```diff
@@ -112,3 +112,97 @@ func TestSqsQueue(t *testing.T) {
 		})
 	}
 }
+
+func TestSqsQueuePolicySupplier_Resources(t *testing.T) {
```
Comment (wbeuil) [inline pkg/remote/sqs_scanner_test.go:116]: Just `TestSQSQueuePolicy`

**Review by wbeuil — CHANGES_REQUESTED** [review]

File: pkg/remote/aws/sqs_queue_policy_enumerator.go (hunk 1 above)
Comment (lotoussa) [reply pkg/remote/aws/sqs_queue_policy_enumerator.go:12]: i have changed Sqs to SQS everywhere in the project on another PR. Let me know if it's too much and you prefer to change only on this PR, but for example on #750 it's still Sqs so i think it's not clean to change only on #749.

File: pkg/remote/sqs_scanner_test.go (hunk 2 above)
Comment (lotoussa) [reply pkg/remote/sqs_scanner_test.go]: done

**Review by wbeuil — APPROVED** [review]

**Pull request merged**